static/app/views/insights/common/components/insightsTimeSeriesWidget.tsx

```diff
@@ -48,18 +51,22 @@ export function InsightsTimeSeriesWidget(props: InsightsTimeSeriesWidgetProps) {
     })) ?? [];
 
   const visualizationProps: TimeSeriesWidgetVisualizationProps = {
-    visualizationType: props.visualizationType,
-    timeSeries: (props.series.filter(Boolean) ?? [])?.map(serie => {
+    plottables: (props.series.filter(Boolean) ?? [])?.map(serie => {
       const timeSeries = convertSeriesToTimeseries(serie);
+      const PlottableDataConstructor =
+        props.visualizationType === 'line'
+          ? Line
+          : props.visualizationType === 'area'
+            ? Area
+            : Bars;
```

Comment:
Minor comment because I see this is only used in two places, but maybe we would want to extract this out into a separate util/factory?

Reply:
Yes, I agree! I'll do that in a follow-up, when I integrate the `PlotType` type